doc/rst/language/archivedSpecs.rst

```diff
@@ -44,7 +44,7 @@ Spec *<version>* (Chapel *<release in which specification was made available>*)
 
 
 
-.. _Language Specification:     https://chapel-lang.org/docs/latest/language/spec.html
+.. _Language Specification:     https://chapel-lang.org/docs/language/spec.html
```

Comment:
This could arguably be a cross-reference to the current version's spec rather than latest using a `:ref:`

Reply:
I'm not necessarily opposed to that, except that the text just above it says "Latest Spec Version" which seems like it could be unnecessarily confusing if it were pointing to the latest version as of some past release date.  (It's also excerbated by the fact that the link to this page scrolls down just enough, in my Chrome browser at least, that it hides the little color-coded version menu).  Try this link to see what I mean, both in terms of the scrolling and what the wording implies if you just found yourself on this page:  https://chapel-lang.org/docs/1.14/language/archivedSpecs.html#chapel-archived-specs  

Thoughts?